src/components/Inventory/view/BatchDetailsModal.tsx

```diff
@@ -19,7 +19,7 @@ export interface BatchDetailsModalProps {
   onClose: () => void;
   reload: () => void;
   organization: ServerOrganization;
-  selectedBatch: Batch | undefined;
+  selectedBatch: any;
```

Comment:
selectedBatch is not of type `Batch` - it has search response type data not GET type data